corehq/apps/settings/forms.py

```diff
@@ -54,13 +54,11 @@ def __init__(self, user, *args, **kwargs):
                     data_bind="value: password, valueUpdate: 'input'",
                 ),
                 'new_password2',
-                hqcrispy.FormActions(
```

Comment:
Out of curiosity, why was this removed? Is it no longer needed in B5?

Reply:
It's still in B5, and it's still in our docs for two-column forms ([here](https://www.commcarehq.org/styleguide/b5/organisms/forms/#crispy-forms-simple)). I've been pulling it out of vertical forms as a simplification. With the submit button now all the way to the left, it's naturally what the user sees as they finish the form.

I thought removing form actions was our standard for vertical forms, but now I'm realizing that we don't have a documented standard, I've been working off of the [maintenance alerts page](https://staging.commcarehq.org/alerts/) (one of the first pages that was migrated to B5, which uses vertical forms).

FYI @biyeun in case you have thoughts on what our standards for vertical forms should be.